pkgs/development/python-modules/qubes-core-admin-client/0003-fix-do-not-crash-on-pool-with-no-volumes.patch

```diff
@@ -0,0 +1,25 @@
+From 2448b386505af8540f21b0f81275f45776070f8c Mon Sep 17 00:00:00 2001
+From: Yaroslav Bolyukin <iam@lach.pw>
+Date: Sat, 14 Sep 2024 00:09:19 +0200
+Subject: [PATCH 3/3] fix: do not crash on pool with no volumes
```

Comment:
TODO: Submit upstream

This is not an issue on QubesOS, because in qubes you have OOTB setup, and pools are never empty.

This is not the case on NixOS, on Nix you need to install distributions after.

Reply:
Upstream PR: https://github.com/QubesOS/qubes-core-admin-client/pull/311

Reply:
Merged upstream